packages/wdio-selenium-standalone-service/src/launcher.ts

```diff
@@ -53,7 +53,7 @@ export default class SeleniumStandaloneLauncher {
         this.skipSeleniumInstall = Boolean(options.skipSeleniumInstall)
 
         // simplified mode
-        if (options.drivers && Object.keys(options.drivers).length > 1) {
+        if (options.drivers && Object.keys(options.drivers).length > 0) {
```

Comment:
Maybe we move `0` to a const to name it property which helps to understand its meaning.

Reply:
I'll add a test and add function with meaningful name